variables.tf

```diff
@@ -172,6 +172,13 @@ variable "identity_type" {
   }
 }
 
+variable "idle_timeout_in_minutes" {
+  type        = number
+  description = "(Optional) Desired outbound flow idle timeout in minutes for the cluster load balancer. Must be between 4 and 120 inclusive."
+  default     = 30
+  nullable    = false
```

Comment:
According to the [provider schema](https://github.com/hashicorp/terraform-provider-azurerm/blob/307e6478cf507c41301413475d77574618031db5/internal/services/containers/kubernetes_cluster_resource.go#L797-L802), the `idle_timeout_in_minutes` argument is optional, which means we can set it to `null`.